primo/data_parser/well_data.py

```diff
@@ -935,10 +1007,16 @@ def _process_data(self, metrics: SetOfMetrics):
 
     def _process_dac_data(self):
         """
-        processes the dac data
+        processes the DAC data
         """
         self._append_fed_dac_data()
-        return
+        metrics = self.config.impact_metrics
+        metric = [obj for obj in metrics if obj.name == "fed_dac"][0]
+        weight = metric.effective_weight
+        metric.data_col_name = "Disadvantaged Score"
+        self.data[metric.score_col_name] = (
+            self.data[metric.data_col_name] * weight / 100
+        )
```

Comment:
Shall we use the same formula ((score - min_score) / (max_score - min_score)) * effective_weight to keep it consistent with the rest of the calculations? If yes, then this method can be removed. `compute_priority_scores` method will take care of the calculation. We just need to add
`self.config.impact_metrics.fed_dac.data_col_name` = "Disadvantaged Score" in the `self._append_fed_dac_data` method.

Reply:
I think that may be non ideal for scenarios such as a state where every well is in a disadvantaged area: Or the min_score is 50 and max is 100. Users would still like to know that the wells with partially disadvantaged status are identified as such and not given a score of 0

Reply:
Okay. sounds good. We can keep the current approach. Suggest simplifying the metric object identification:
```python
metrics = self.config.impact_metrics.fed_dac
```

Reply:
We just have to keep track of the priority scores that are calculated differently and if there is a major change introduced to how we have been calculating them (e.g., percentile based distribution) then we will have to ensure that we check these locations as well and update as necessary